qiskit/tools/backends/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
```

Comment:
```suggestion
# Copyright 2019, IBM.
```

Reply:
The copyrights are changing today anyway so this will change.